cl_dll/hud_customtimer.cpp

```diff
@@ -0,0 +1,61 @@
+#include "hud.h"
+#include "cl_util.h"
+
+DECLARE_COMMAND(m_CustomTimer, CustomTimer);
+
+int CHudCustomTimer::Init()
+{
+	HOOK_COMMAND("customtimer", CustomTimer);
+
+	m_iFlags = 0;
+
+	gHUD.AddHudElem(this);
+	return 0;
+}
+
+int CHudCustomTimer::VidInit()
+{
+	m_iFlags &= ~HUD_ACTIVE;
+	m_iTimerPos = gHUD.m_scrinfo.iCharHeight * 4;
```

Comment:
any reason why this is saved into a member variable? and not just computed inside Draw()